app/graphql/types/system.rb

```diff
@@ -47,7 +47,12 @@ class System < Types::BaseObject
       argument :profile_id, String, 'Filter results by profile ID',
                required: false
     end
-
+    field :rules_failed_severity, [::Types::Rule], null: false do
+      argument :policy_id, ID, 'Filter results by system ID',
+               required: false
+      argument :severity, [String], 'Severity levels of failed rules',
+               required: false
```

Comment:
Same here, isn't it required? Or what happens if the severity is unset with the code below?